authentication/models.py

```diff
@@ -33,9 +38,88 @@ class CustomUser(AbstractBaseUser, PermissionsMixin):
     is_active = models.BooleanField(default=True)
     is_staff = models.BooleanField(default=False)
 
+
     USERNAME_FIELD = "email"
     REQUIRED_FIELDS = ["password", "first_name", "surname", "phone_number"]
     objects = CustomUserManager()
 
     def __str__(self):
         return f"{self.first_name} {self.surname} {self.email}"
+    
+    def get_auth_data(self):
+        return {'email': self.email,
+                'password': self.password,
+                'first_name': self.first_name,
+                'surname': self.surname,
+                'user_id': self.user_id,
+                'is_verified': self.is_verified,
+                'is_superuser': self.is_superuser}
+    
+
+class Companies(models.Model):
+    company_id = models.BigAutoField(primary_key=True, unique=True, auto_created=True)
```

Comment:
Explicit using primary key field with ```auto_created=True``` is redundant since Django automatically adds an AutoField as the primary key if none is specified.

Reply:
The option removed